ui/analyse/src/treeView/columnView.ts

```diff
@@ -141,14 +142,26 @@ function renderMainlineMoveOf(ctx: Ctx, node: Tree.Node, opts: Opts): VNode {
       attrs: { p: path },
       class: classes,
     },
-    moveView.renderMove(ctx, node, notationStyle(ctx.ctrl.data.game.variant.key))
+    moveView.renderMove(
+      { variant: ctx.ctrl.data.game.variant, ...ctx }, node, notationStyle(ctx.ctrl.data.game.variant.key))
   );
 }
 
 function renderVariationMoveOf(ctx: Ctx, node: Tree.Node, opts: Opts): VNode {
+  const variant = ctx.ctrl.data.game.variant;
+  const notation = notationStyle(variant.key);
   const withIndex = opts.withIndex || node.ply % 2 === 1,
     path = opts.parentPath + node.id,
-    content: MaybeVNodes = [withIndex ? moveView.renderIndex(node.ply, true) : null, fixCrazySan(node.san!)],
+    content: MaybeVNodes = [
+      withIndex ? moveView.renderIndex(node.ply, true) : null,
+      // TODO: the || '' are probably not correct
+      moveFromNotationStyle(notation)({
+        san: fixCrazySan(node.san || ''),
+        uci: node.uci || '',
+        fen: node.fen,
+        prevFen: ''
```

Comment:
prevFen is currently required for shogi due to only uci being available. Empty string will work afaik for the other defaults, but if you are asking for san and give an empty string for it it's not going to work anyways.....

Reply:
Yeah, maybe we can jump on a zoom call to talk through this to make sure I understand?